include/apps/tools/OswAppToolsApp.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <osw_hal.h>
+#include <osw_ui.h>
+#include "osw_app.h"
+
+class OswAppToolsApp : public OswApp {
+  public:
+    OswAppToolsApp(void) {
+        ui = OswUI::getInstance();
+    };
+    virtual void setup() override;
+    virtual void loop() override;
+    virtual void stop() override;
```

Comment:
Virtual in parent class -> no need to respecify them in an abstract child class.

Reply:
yes, I will take a look at that ..

Reply:
I just copied what was already there .. I will have to look at it